### ISSUE TITLE
different formats for rural/non-rural

### ISSUE THREAD

**Issue by freyfogle** [OpenCageData/address-formatting#4]
Am coming acorss countries where in larger cities it seems just the city is used, but in rura areas town and province (or equiv) is used.

**Comment by StephanGeorg** [comment]
...like in italy.

**Comment by freyfogle** [comment]
yes, do you have any test cases we could start trying to work with? Unfortunately I don't see a simple solution here as all we have to work with is the input components, not real geographic information. All ideas or approaches welcome.

**Comment by StephanGeorg** [comment]
Honestly no. I'm processing a lot of italian data but can't find regularity when to use "province code" or not. 

According to the [official docs](https://www.poste.it/standard_composizione_indirizzi.pdf) I would say `{{state_district}}`  should be used by default.
